common/models/user.js

```diff
@@ -269,6 +269,14 @@ module.exports = function(User) {
    */
 
   User.logout = function(tokenId, fn) {
+    if(!tokenId) {
+      process.nextTick(function() {
+        var err = new Error('access token is required to logout');
+        err.status = 400;
+        fn(err);
+      });
+      return;
+    }
     this.relations.accessTokens.modelTo.findById(tokenId, function(err, accessToken) {
```

Comment:
Can this just be `this.accessTokens.destroyById(tokenId, function(err) {...}`? I also wondering if we need to provide feedback if the access_token is invalid.